README.rst

```diff
@@ -10,3 +10,20 @@ Coaster: common patterns for Flask apps
 
 Coaster contains functions and db models for recurring patterns in Flask
 apps. Documentation at http://coaster.readthedocs.org/
+
+### Run tests
+
+You can also look at `.travis.yml` for instructions on how to run tests.
```

Comment:
Remove this line. We don't want to confuse people with Travis-specific instructions.